frontend/src/views/SecretOverviewPage/components/ManageSecretModals/DeleteSecretModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { type FC } from "react"
+import { Modal, ModalContent, Button } from "@app/components/v2";
+
+type Props = {
+  isModalOpen: boolean;
+  handleDeleteSecret: () => void;
+  toggleModal: () => void;
+};
+
+export const DeleteSecretModal: FC<Props> = ( { isModalOpen, toggleModal, handleDeleteSecret }) => {

```

Comment:
Can you reuse the existing one we already have when we go to delete a secret inside of the dashbaord?
<img width="1823" alt="CleanShot 2024-07-26 at 13 51 08@2x" src="https://github.com/user-attachments/assets/013bdfbc-958a-41da-ad30-25a179462847">

Reply:
I've updated it.